Assets/Scripts/ScripatableObjects/TankScriptableObject.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TankBattle.Tank;
+
+[CreateAssetMenu(fileName = "TankScriptableObject", menuName = "ScriptableObject/TankScriptableObject", order = 0)]
+public class TankScriptableObject : ScriptableObject {
```

Comment:
namespace

Reply:
moved tank and shell scriptable objects into respected folders and used the same namespace as the MVC